js/app.js

```diff
@@ -74,13 +75,16 @@ function buildBlogSummary(blog, modularContent)
     const title = firstOrDefaultValue(blog, "title") || "unknown";
     blogSummary.querySelector(".summary-title").textContent = title;
 
-    blogSummary.querySelector(".summary-blog-post").dataset.item_id =
+    const blogPost = blogSummary.querySelector(".summary-blog-post");
+    blogPost.dataset.item_id =
         blogSummary.querySelector(".blog-preview-toggle").dataset.item_id = blog.system.id;
 
     const blogMediaImageURL = firstOrDefaultValue(blog, "blog_media___image", "url");
 	if (blogMediaImageURL)
 	{
-		blogSummary.querySelector(".blog-image").src = blogMediaImageURL;
+        const blogImage = document.importNode(blogImageTemplate.content, true);
```

Comment:
Something strange going on with whitespace (indentation) here. If it's a problem with tabs Vs spaces we should standardise (probably on spaces)

Reply:
Looks fine in the code, this is the removed line, but not sure why it is showing like that...